arch/risc-v/src/common/riscv_misaligned.c

```diff
@@ -439,13 +514,35 @@ static bool decode_insn(uintptr_t *regs, riscv_insn_ctx_t *ctx)
 
         break;
 
-      /* TODO: Handle float load / store instruction */
-
+#ifdef CONFIG_ARCH_FPU
       case INSN_FLW:
       case INSN_FLD:
```

Comment:
Do you have plans to support `INSN_FLQ` / `INSN_FSQ`

Reply:
Yes if any device support it